test/tests/meshgenerators/generated_mesh_generator/tests

```diff
@@ -10,4 +10,15 @@
     mesh_mode = 'REPLICATED'
     recover = false
   [../]
+  [./generated_mesh_generator_mesh_type_test]
+    prereq = generated_mesh_generator_test
+    type = 'RunException'
+    input = 'generated_mesh_generator.i'
+    cli_args = '--mesh-only Mesh/type=GeneratedMesh'
+    expect_err = 'Mesh Generators present but the \[Mesh\] block is set to construct a "GeneratedMesh" mesh, which does not use Mesh Generators in constructing the mesh.'
+    requirement = 'MOOSE shall generate a warning if Mesh Generators are used with a mesh type that does not accept them'
+    design = 'meshgenerators/MeshGeneratorMesh.md'
```

Comment:
Hehe - these are complicated names. You put the pattern "BAB" in instead of "ABA" here!

Reply:
I think it's right now.  Thanks!